### PR TITLE
Feat/cs 2535 topic view

### PR DESCRIPTION
**Pull request by RamaChandiniDevarapalli**

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { TopicTableItem } from './topicTableItem';
+import { TopicItem } from '@store/comment/model';
+
+import { HeaderFont, TableDiv } from '../styled-components';
+
+export interface TopicTableProps {
+  topics: TopicItem[];
+  onDeleteTopic?: (Topic) => void;
+}
+export const TopicListTable: FunctionComponent<TopicTableProps> = ({ topics, onDeleteTopic }) => {
```
Comment (solittlework) [inline apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicsTable.tsx:12]: Not sure why we need the FunctionComponent here.

File: apps/tenant-management-webapp/src/app/pages/admin/services/comment/comments/addCommentModal.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+interface TopicModalProps {
+  topic: TopicItem;
+  open: boolean;
+  type: string;
+  onCancel?: () => void;
+  onSave: (comment: Comment) => void;
+}
+
+export const AddCommentModal: FunctionComponent<TopicModalProps> = ({
+  topic,
```
Comment (solittlework) [inline apps/tenant-management-webapp/src/app/pages/admin/services/comment/comments/addCommentModal.tsx:19]: I think we can have better way to de-couple this modal with the rest of the code. I think we do not need to control the modal open by redux.

File: apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicTableItem.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+export const TopicTableItem: FunctionComponent<TopicTableItemProps> = ({
+  id,
+  topic,
+  onDeleteTopic,
+}: TopicTableItemProps) => {
+  const [showDetails, setShowDetails] = useState(false);
+
+  const onDeleteComment = (comment) => {
+    console.log(comment);
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicTableItem.tsx:21]: please remove console.log

File: apps/tenant-management-webapp/src/app/store/comment/action.ts [hunk 4]
```diff
@@ -9,6 +9,21 @@ export const UPDATE_COMMENT_TOPIC_TYPE_SUCCESS_ACTION = 'comment/UPDATE_COMMENT_
 export const DELETE_COMMENT_TOPIC_TYPE_ACTION = 'comment/DELETE_COMMENT_TOPIC_TYPE_ACTION';
 export const DELETE_COMMENT_TOPIC_TYPE_SUCCESS_ACTION = 'comment/DELETE_COMMENT_TOPIC_TYPE_SUCCESS_ACTION';
 
+export const ADD_TOPIC_REQUEST = 'ADD_TOPIC_REQUEST';
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/store/comment/action.ts:12]: Please consist with another definition for action constants.

**Comment by athena-chen-chen** [comment]
Please fix those lint warning, thanks!

File: apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicsTable.tsx (hunk 1 above)
Comment (solittlework) [reply apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicsTable.tsx]: Talked with Rama and we can improve this later.

File: apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicTableItem.tsx (hunk 3 above)
Comment (solittlework) [reply apps/tenant-management-webapp/src/app/pages/admin/services/comment/topics/topicTableItem.tsx]: I think the issue has been resolved.

**Review by solittlework — APPROVED** [review]

**Review by athena-chen-chen — APPROVED** [review]

**Pull request merged**